pom.xml

```diff
@@ -13,7 +13,7 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.509.4</version>
+		<version>1.609.1</version>
```

Comment:
Yeah, but upgrading to the new plugins parent POM usually requires more fixed here and there. Let me do that a separate PR.